app/src/main/java/org/schabi/newpipe/BaseFragment.java

```diff
@@ -121,7 +121,7 @@ public void setTitle(final String title) {
 
     protected FragmentManager getFM() {
         return getParentFragment() == null
-                ? getFragmentManager()
-                : getParentFragment().getFragmentManager();
+                ? getParentFragmentManager()
+                : getParentFragment().getChildFragmentManager();
```

Comment:
I'm not quite sure I understand this change. The behaviour here has changed.

Reply:
Now that I think about it, I'm not quite sure myself. I'll revert this change.

Reply:
Yeah, I don't understand either what exactly this piece of code does. It was added in 8dde25532a06908c813b8b9909dc7d95f455960d, which doesn't provide any explanation. `getFragmentManager()` is just a deprecated `null`able alias of `getParentFragmentManager()`, isn't it? In another PR I think you should therefore probably replace `getFM()` everywhere with just `getParentFragmentManager()`, as it seems like just a workaround for the `null`ability.